packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -243,6 +243,196 @@ class _LinearProgressIndicatorPainter extends CustomPainter {
 /// The minimum height of the indicator can be specified using [minHeight].
 /// The indicator can be made taller by wrapping the widget with a [SizedBox].
 ///
+/// {@tool sample --template=stateless_widget_scaffold}
+///
+/// This example shows a [LinearProgressIndicator] with a fixed value.
+///
+/// ![https://flutter.github.io/assets-for-api-docs/assets/material/linear_progress_indicator_fixed_value.png]
+///
+/// ```dart
+/// Widget build(BuildContext context) {
+///   return Padding(
+///     padding: const EdgeInsets.all(20.0),
+///     child: Column(
+///       mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+///       children: [
+///         Text(
+///           'Linear progress indicator with a fixed value',
+///           style: const TextStyle(fontSize: 20),
+///         ),
+///         LinearProgressIndicator(
+///           value: 0.5,
+///           semanticsLabel: 'Linear progress indicator',
+///         ),
+///       ],
+///     ),
+///   );
+/// }
+/// ```
+/// {@end-tool}
+///
+/// {@tool dartpad --template=stateless_widget_scaffold}
+///
+/// This example shows a [LinearProgressIndicator] with a fixed color.
+///
+/// ```dart
+/// Widget build(BuildContext context) {
+///   return Padding(
+///     padding: const EdgeInsets.all(20.0),
+///     child: Column(
+///       mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+///       children: [
+///         Text(
+///           'Linear progress indicator with a fixed color',
+///           style: const TextStyle(fontSize: 20),
+///         ),
+///         LinearProgressIndicator(
+///           valueColor: AlwaysStoppedAnimation<Color>(Colors.blueGrey),
+///           semanticsLabel: 'Linear progress indicator',
+///         ),
+///       ],
+///     ),
+///   );
+/// }
+/// ```
+/// {@end-tool}
+///
+/// {@tool sample --template=stateless_widget_scaffold}
+///
+/// This example shows a [LinearProgressIndicator] with
+/// a fixed color, fixed value and a background color.
+///
+/// ![https://flutter.github.io/assets-for-api-docs/assets/material/linear_progress_indicator_background_color.png]
```

Comment:
Same here, there needs to be a file to generate this asset here: https://github.com/flutter/assets-for-api-docs/tree/master/packages/diagrams/lib/src

Reply:
You could also take this image out if you want.